test/onnx/exporter/test_building.py

```diff
@@ -0,0 +1,147 @@
+# Owner(s): ["module: onnx"]
+"""Unit tests for the _building module."""
+
+from __future__ import annotations
+
+import numpy as np
+import onnxscript
+from onnxscript import ir
+
+import torch
+from torch.onnx._internal.exporter import _building, _tensors
+from torch.testing._internal import common_utils
+
+
+class TestOpRecorder(common_utils.TestCase):
+    def setUp(self):
+        self.opset_version = 17
+        self.opset = onnxscript.values.Opset("", self.opset_version)
+        self.recorder = _building.OpRecorder(opset=self.opset, constant_farm={})
+
+        self.model = ir.Model(
+            graph=ir.Graph(
+                [],
+                [],
+                nodes=[],
+                opset_imports={
+                    "": self.opset_version,
+                },
+                name="main_graph",
+            ),
+            ir_version=9,
+            producer_name="pytorch",
+            producer_version=torch.__version__,
+        )
+
+    def test_skippable_castlike(self):
```

Comment:
I would refine the test name to explain a little better (e.g. "redundant castlike is omitted from graph" or something more succinct)

Reply:
Done